### PR TITLE
Adds pysgrid library

### PR DESCRIPTION
**Pull request by daf**

### CODE REVIEW

File: pysgrid/bld.bat [hunk 1]
```diff
@@ -0,0 +1 @@
+"%PYTHON%" setup.py install --single-version-externally-managed  --record record.txt
```
Comment (ocefpaf) [inline pysgrid/bld.bat]: You have to remove the ` --single-version-externally-managed --record record.txt` here too.

File: pysgrid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    git_url: https://github.com/sgrid/pysgrid.git
+    git_tag: "0.0.4-beta"
+
+build:
+    number: 1
+
+requirements:
+    build:
+        - python
+        - setuptools
```
Comment (ocefpaf) [inline pysgrid/meta.yaml]: pysgrid does not use setuptools :unamused:

Comment (kwilcox) [inline pysgrid/meta.yaml]: I'll fix pysgrid

**Comment by kwilcox** [comment]
https://github.com/sgrid/pysgrid/pull/51

File: pysgrid/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package:
+    name: pysgrid
+    version: "0.0.4b"
+
+source:
+    git_url: https://github.com/sgrid/pysgrid.git
+    git_tag: "0.0.4-beta"
+
+build:
+    number: 1
```
Comment (ocefpaf) [inline pysgrid/meta.yaml]: Can you make this `0`?

**Comment by ocefpaf** [comment]
@kwilcox Do you want me to wait for sgrid/pysgrid#51?

**Comment by kwilcox** [comment]
@ocefpaf might as well merge this if it works, @daf can always send another PR when `pygrid` starts uses `setuptools`.

**Pull request merged**